Description: Fix Python 3.5 compat
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-02-05

--- python-trollius-2.0.orig/trollius/futures.py
+++ python-trollius-2.0/trollius/futures.py
@@ -422,9 +422,6 @@ class Future(object):
                 result = other.result()
                 self.set_result(result)
 
-    if compat.PY35:
-        __await__ = __iter__ # make compatible with 'await' expression
-
 if events.asyncio is not None:
     # Accept also asyncio Future objects for interoperability
     _FUTURE_CLASSES = (Future, events.asyncio.Future)
